milli/src/documents/builder.rs

```diff
@@ -135,6 +136,25 @@ impl<W: Write> DocumentsBatchBuilder<W> {
                             }
                         }
                     }
+                    AllowedType::Boolean => {
+                        let trimmed_value = value.trim();
+                        if trimmed_value.is_empty() {
```

Comment:
Why not trim above the match once for all? 😄

Reply:
Because we don’t need to trim strings, and since strings are probably the most used branch by far (since everything defaults to string if you forget to type your csv), I feel like it’s best to avoid any useless computation

Reply:
Hum, yes, but it doesn't take much time to skip whitespace on the front and the back of a string, and, more importantly, I am pretty sure it will be optimized and reordered by the compiler.

Can we put it above this match? 😃